docs/source/examples.daisy_chain_gap.ipynb

```diff
@@ -0,0 +1,746 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "all_smiles = [\n",
+    "    'CC1=CCC=CC(C)=C1C(C)C', \n",
+    "    'CC1(c2ccc(CC3CC=CC3)cc2)CC1', \n",
+    "    'C#CC[C@@H](CCC=C(C)C)C1CC1', \n",
+    "    'Cc1ccccc1CCCC1=CCCCC1', \n",
+    "    'C=CC1=CC[C@@H]2C[C@H]1C2(C)C', \n",
+    "    'C1=CCC(Cc2ccc(CC3CC3)cc2)C1', \n",
+    "    'C1=CC(c2ccccc2)=CCC1', \n",
+    "    'C/C=C/CCCC[C@H](C)C(C)(C)C', \n",
+    "    'C=C[C@@H]1C/C=C/CCCCCCCC1', \n",
+    "    'C[C@H](CC(C)(C)C)[C@@H](C)C(C)(C)C', \n",
+    "    'CC/C=C\\\\C[C@@H](C)c1cccc(C)c1C', \n",
+    "    'C=C1CC2c3ccccc3C1c1ccccc12']\n",
+    "\n",
+    "outputs = OutputSpec(\"1.ch.rdkit.xyz\")\n",
+    "inputs = smiles.run(all_smiles, outputs=outputs)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run Molecular Dynamics simulation\n",
+    "\n",
+    "We run the MD at 300 K with an NVT Berendsen thermostat. \n",
+    "\n",
+    " TODO: how do we manage remote execution examples code-testing-wise?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Creating job md_chunk_0\n",
+      "ExPyRe md_chunk_0 constructor done stage in files 1675187004.1068766\n",
+      "Starting job for md_chunk_0_wwv-BW3fpA41qkNP23A5vAqOoZXuw6OXgeKRWgt0CQ8=_s95p1ub6\n",
+      "Gathering results for md_chunk_0_wwv-BW3fpA41qkNP23A5vAqOoZXuw6OXgeKRWgt0CQ8=_s95p1ub6 remote 205293\n",
+      "Waiting for job md_chunk_0_wwv-BW3fpA41qkNP23A5vAqOoZXuw6OXgeKRWgt0CQ8=_s95p1ub6 up to 3600 s: \n",
+      "qqrr\n",
+      "Running wfl.generate.md._sample_autopara_wrappable with num_python_subprocesses=4, num_inputs_per_python_subprocess=1\n"
+     ]
+    }
+   ],
+   "source": [
+    "outputs = OutputSpec(\"2.ch.rdkit.md.traj.xyz\")\n",
+    "\n",
+    "md_params = {\n",
+    "    \"steps\": 80,\n",
+    "    \"dt\": 0.5,  # fs\n",
+    "    \"temperature\": 300,  # K\n",
+    "    \"temperature_tau\": 500,  \n",
+    "    \"results_prefix\": \"xtb_\",\n",
+    "    \"traj_step_interval\": 5}\n",
+    "\n",
+    "remote_info = {\n",
+    "    \"sys_name\" : \"local\", \n",
+    "    \"job_name\" : \"md\", \n",
+    "    \"resources\" : { \n",
+    "        \"max_time\" : \"15m\",\n",
+    "        \"num_cores\" : 4,\n",
+    "        \"partitions\" : \"any$\"}, \n",
+    "    \"partial_node\": True,\n",
+    "    \"check_interval\": 5,\n",
+    "    \"num_inputs_per_queued_job\" :20,\n",
+    "    \"pre_cmds\" :[\"conda activate dev\"]\n",
+    "}\n",
+    "\n",
+    "# set to None for github testing purposes\n",
+    "remote_info=None\n",
+    "\n",
+    "inputs = md.sample(\n",
+    "    inputs=inputs, \n",
+    "    outputs=outputs,\n",
+    "    calculator=xtb_calc,\n",
+    "    autopara_info = AutoparaInfo(\n",
```

Comment:
I intended to code these up so that you could pass a dict of constructor kwargs and not have to actually import `AutoparaInfo`. We should think about whether that's enough of a convenience to be worth using routinely, or if we always want to import the object definition and construct it explicitly.

Reply:
I don't have strong feelings about it.  I suppose we could encourage the safer approach which is to use the classes.